client/gutenberg/extensions/visited/index.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import edit from './components/edit';
+import save from './components/save';
+import { CRITERIA_AFTER, DEFAULT_THRESHOLD } from './constants';
+import './editor.scss';
+import './view.scss';
+
+export const name = 'visited';
+export const settings = {
+	attributes: {
+		criteria: {
+			type: 'string',
+			default: CRITERIA_AFTER,
+		},
+		threshold: {
+			type: 'number',
+			default: DEFAULT_THRESHOLD,
+		},
+	},
+	category: 'jetpack',
+	description: __(
```

Comment:
Let's add some keywords here as well. So that users are able to find the block easier.